.github/workflows/release.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
           KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
           KEY_STORE_PASSWORD: ${{ secrets.KEY_STORE_PASSWORD }}
-        run: ./gradlew publishReleaseApk
+        run: ./gradlew publishReleaseBundle ; ./gradlew publishReleaseApk
```

Comment:
Also can you tell which out of these two is going to be published at playstore? I think we should separate out these two commands, otherwise maybe it'll push the result of latst command? Again, to avoid grey area, I'll re-iterate about the simulation in the PR

Reply:
Yes, we can separate those commands. This is making confusing.